code/game/area/areas/ruins/space_areas.dm

```diff
@@ -45,19 +45,17 @@
 
 //DJSTATION
 /area/ruin/space/djstation
-	name = "\improper Ruskie DJ Station"
+	name = "\improper Soviet DJ Station"
 	icon_state = "DJ"
-	there_can_be_many = TRUE
 
 /area/ruin/space/djstation/solars
-	name = "\improper Ruskie DJ Station Solars"
+	name = "\improper Soviet DJ Station Solars"
 	icon_state = "DJ"
 
 //Methlab
 /area/ruin/space/methlab
 	name = "\improper Abandoned Drug Lab"
 	icon_state = "green"
-	there_can_be_many = TRUE
```

Comment:
why would you do that without changing `/datum/map_template/ruin/space/druglab`'s `allow_duplicates` to `false`?
and why would you ever touch an other space ruin than dj

Reply:
![image](https://github.com/user-attachments/assets/40e92826-5e84-4350-ac1a-8b3b7b81ad96)

The default for ruins is TRUE, no need to be repeated there.

It is minor/close enough to end up touching this one too. Personally if Fordoxia ended up removing the other two that have it in this file then it would be a bit out of scope.

Reply:
yeah i see it now, thx for pointing out, thought the default is still false

Reply:
> why would you do that without changing `/datum/map_template/ruin/space/druglab`'s `allow_duplicates` to `false`? and why would you ever touch an other space ruin than dj

I may be stupid.